ParisBNF/abb/BNFabb207.xml

```diff
@@ -0,0 +1,278 @@
+                        </physDesc>
+                    </msPart>
+                                
+                                
+
+                    <msPart xml:id="p2">
+                        <msIdentifier/>
+                        <msContents>
+                            <summary/>
+                            
+                                <msItem xml:id="p2_i1">
+                            <locus from="41r"/>
+                            <title type="complete">Prayer for gathering all 81 biblical books</title>
+                           </msItem>
+
+                                <msItem xml:id="p2_i2">
+                               <locus from="41r" to="44v" cert="medium"/>
+                            <locus from="55r" to="56v"/>
+                            <title type="complete">Minor protective texts</title>
+                            </msItem> 
+                            
+                            <msItem xml:id="p2_i3">
+                                <locus from="45r" to="54v" cert="medium"/>                              
+                                <title type="complete">Magic ʾasmāt names that God gave 
+                                    to Moses, to liberate his people from servitude</title>
+                            </msItem>                            
+                                           
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                           <material key="parchment"/>
+                        </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="15">15</measure>
+                                    <locus from="41" to="56"></locus>
+                    
+                                    <dimensions type="leaf" unit="mm">
+                                        <height>30</height>
+                                        <width>30</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="good"/>
+                              </supportDesc>
+
+                             <layoutDesc>
+                                <layout columns="2" writtenLines="15">
+                                  <locus from="45r" to="54v"/>                                  
+                                </layout>
+                             </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                           <handNote xml:id="h2" script="Ethiopic">
+                              <seg type="script">18th-century script</seg>
+                              <seg type="ink">Black, red</seg>
+                               <desc>The script of extremely small size.</desc>
```

Comment:
I suggest either `The script is of extremely small size.` or `Script of extremely small size.`

Reply:
ok